src/patternfly/components/InlineEdit/inline-edit.scss

```
@@ -1,6 +1,6 @@
 // @debug $inline-edit; // check your variable names located in src/patternfly/sass-utilities/component-namespaces.scss
 
-.#{$inline-edit} {
+:root, .#{$inline-edit} {
```

Can we use the `:where` pseudo here?

```suggestion
:root, 
:where(.#{$inline-edit}) {
```